### ISSUE TITLE
[rare] tag for kanji surface forms.

### ISSUE THREAD

**Issue by jimmubreen** [JMdictProject/JMdictIssues#9]
Perhaps best to see Marcus' comments on the べきではない entry for 2020-04-03. (https://www.edrdg.org/jmdictdb/cgi-bin/entr.py?svc=jmdict&sid=&q=2027080)

The idea is that for an entry such as 可き/べき, there should be a [rare] tag available for the rarely-used 可き form to reinforce the [uk] tag on the kana form.

**Comment by JMdictProject** [comment]
I'm proposing to add an "rK" tag for these cases.

**Comment by JMdictProject** [comment]
Done

**Comment by Marcusjmdict** [comment]
I've added the tag to some 200 entries: a start.

**Comment by Marcusjmdict** [comment]
A difference of opinion regarding the application of [rK] came up in the ０歳 entry:
https://www.edrdg.org/jmdictdb/cgi-bin/entr.py?svc=jmdict&sid=&q=1965500.1
I added the tag to three of the kanji surface forms because low numbers in the ngrams:
０歳； ０才； 〇歳； ゼロ歳 ； ゼロ才[rK] ； 零歳 [rK] ； 零才 [rK]
Robin responded:

> I don't think we can tag ゼロ才 as rK when it has the same kanji as ０才. It's an uncommon form but not specifically because of the kanji.

I responded :

> I don't think it makes sense to use rK to mean 
> "this form contains rarely used kanji" rather 
> than "this kanji field surface form is rarely 
> used". The point of it as I see it is to tell 
> learners they're very unlikely to ever come 
> across that form and don't need to bother 
> learning it. Or to tell a learner's dictionary 
> app to hide it entirely.

On the [entry ref guide page](https://www.edrdg.org/jmdictdb/cgi-bin/edhelp.py?svc=jmdict&sid=#kw_rinf) rK is explained as "rarely-used kanji form", not "rare kanji", which I think aligns with what I'm suggesting here. If we consider the purpose of introducing the tag, and want it to be as useful as possible for those purposes, I think forms like ゼロ才 should be tagged as rK.

Also it might be useful to have some kind of rough guideline for when exactly to apply the tag. Most of the entries where I added it, the ratio of the rare form vs the most common form was below 3%. I'm thinking around there is an OK place to draw the line? Or maybe less than 5%? It doesn't have to be a super strict rule of course, the ngrams aren't the end-all of figuring out the usage and they're not infallible.

**Comment by Marcusjmdict** [comment]
@jimmubreen , could you reopen this? Or should I start a new issue?

**Comment by JMdictProject** [comment]
OK. Reopened.

**Comment by JMdictProject** [comment]
When we first discussed and included the "rK" tag I had in mind the sorts of multi-kanji constructions that are the real outliers. Ateji forms of place names, such as 羅府 for Los Angeles are good examples. I hadn't really thought of forms such as 零歳 and 零才 although considering they are much rarer than 0歳 maybe they fit in. And I don't really think it applies to ゼロ歳 or ゼロ才 because while the compound is not that common, the kanji in that role is, of course, very common. To me, it's not a "kanji form" as such.

We probably need to chew this over a bit before we get to a policy/guideline for "rK" tagging. Probably something like 5% of the total kanji form count might be appropriate, Of course, if it were an iK or oK case they should take precedence.

**Comment by Marcusjmdict** [comment]
> We probably need to chew this over a bit before we get to a policy/guideline for "rK" tagging. Probably something like 5% of the total kanji form count might be appropriate, Of course, if it were an iK or oK case they should take precedence.

I've given this some thought while going through a bunch of entries with multiple kanji surface forms. I've generally only applied it to those that get less than 2-3% of the total, sometimes for higher percentages for words that already don't get that many hits in their more common forms. I think that might be more appropriate as a general guideline than 5%, where you might still get some things even a learner might occasionally come across.

**Comment by robinjmdict** [comment]
I agree with Jim that it should be the use of kanji that makes a form "rK". 

I'd describe ゼロ才 as a "rarely used surface form", but if a mixed kanji-kana form is rare, I'd rather we just ignored it or put it in a hidden kanji field (something I've already suggested) so that it's not displayed to users.

**Comment by Marcusjmdict** [comment]
I've sort of settled (in the way I'm tagging kanji) on "less than 1-2% of the most common form/total", with some discretion.

Should we have a [rk] rare kana tag too, by the way? I'm not really aching for one and but for sake of consistency/completeness....

**Comment by robinjmdict** [comment]
How many kana forms (that are worth recording) are rare but not archaic/outdated or irregular? I can't think of cases where an rk tag would be used, except maybe for "correct" readings that have been superseded by 慣用読み. Even then, it's difficult to get a measure of how common a particular reading is.

Regarding rK, has it been agreed that it should only refer to the kanji? I think most people would interpret "rK" to mean "this form is rarely written using this kanji", and therefore the tag shouldn't be added to forms that are rare for some reason other than the kanji that's used. I believe this is also Jim's position.

**Comment by Marcusjmdict** [comment]
I agree there's no burning need for it - it's more that we usually have these things in pairs, and I don't see any particularly good reason to make an exception here. Like you say, it is hard to get a measure of how common each reading is for most words, esp. if we wanted to apply the pretty stringent "1-2% of total usage" guideline I'm currently using for [rK], but then again there's plenty of entries that don't actually have any kanji at all to begin with. Re: "How many kana forms (that are worth recording) are rare but not archaic/outdated or irregular?": we do tend to include
anything that's in any of the major refs, including readings that likely are obscure. One candidate for the tag that comes to mind would be ごうたい for 合体
which seems to only appear in koj and daijs, without any indication it's archaic, but isn't mentioned in any of the others. I think an absolute majority of native speakers would assume any person pronouncing 合体 (with the regularly understood meaning) as ごうたい is half-illiterate, or worse. Other candidates could be readings for common words that appear in the
other references but aren't "変換-able" in standard IMEs. So I don't think it's the case the tag wouldn't get any use at all.

If we had one or preferably two or even three knowledgable native speakers on the team we could run rk candidates by, I think the tag could see plenty
of use (i.e. if they recognized the word and the main reading but not the rk candidate, we'd mark it as [rk]), but those team members are of course
unlikely to materialize any time soon.

Regarding the use of rK, I'd like to get back to that later but for now I'm respecting the current consensus and only applying it as you and Jim have suggested.

**Comment by Marcusjmdict** [comment]
CoulD'Nt we change [rK] to just [rare] and use it for 1) rare kanji surface forms 2) rare okurigana forms 3) rare readings?

(thought of this when looking at this thread:
https://www.edrdg.org/jmdictdb/cgi-bin/entr.py?svc=jmdict&sid=&q=1219650.1 =
)

**Comment by JMdictProject** [comment]
Something like that would probably be useful, and I agree it would be good to have something that applied to readings as well.
We already have [rare] as a little-used misc tag, so it probably should be something different. Alternatively, we could change that other tag.

**Comment by Marcusjmdict** [comment]
I had no idea we already had a [rare] tag. [Only used for 6 entries currently](https://www.edrdg.org/jmdictdb/cgi-bin/srchres.py?svc=jmdict&s1=1&y1=2&t1=&s2=1&y2=1&t2=&s3=1&y3=1&t3=&idtyp=seq&idval=&search=Search&src=1&stat=2&appr=appr&appr=unappr&nfcmp=&nfval=&misc=20&snote=&snotem=0&smtr=&smtrm=0&ts0=&ts1=&refs=&refsm=0&cmts=&cmtsm=0&mt=0&grp=). I feel we should probably phase out that usage either way.

**Comment by JMdictProject** [comment]
This has been quiet for 4 years and we've discussed [rK] else where since. Closing.
